Submitted by: Alexander E. Patrakov
Date: 2005-08-13
Initial Package Version: 2.3.5
Upstream Status: LiveCD specific. Nearly equivalent to adding "localedef" to Makefile.
Origin: Combined RedHat, ALT Linux and Debian changes
Description: Alters the list of supported locales

Adds some ISO-8859-15 locales (RedHat)
Adds ru_RU.CP1251, uk_UA.CP1251 for Windows compatibility (ALT Linux)
Changes "russian" alias to point to the de-facto ru_RU.KOI8-R standard (Debian)

diff -ur glibc-2.3.5.orig/intl/locale.alias glibc-2.3.5/intl/locale.alias
--- glibc-2.3.5.orig/intl/locale.alias	2005-08-13 14:06:12.910337376 +0600
+++ glibc-2.3.5/intl/locale.alias	2005-08-13 14:07:42.435727448 +0600
@@ -63,7 +63,7 @@
 polish          pl_PL.ISO-8859-2
 portuguese      pt_PT.ISO-8859-1
 romanian        ro_RO.ISO-8859-2
-russian         ru_RU.ISO-8859-5
+russian         ru_RU.KOI8-R
 slovak          sk_SK.ISO-8859-2
 slovene         sl_SI.ISO-8859-2
 slovenian       sl_SI.ISO-8859-2
diff -ur glibc-2.3.5.orig/localedata/SUPPORTED glibc-2.3.5/localedata/SUPPORTED
--- glibc-2.3.5.orig/localedata/SUPPORTED	2005-08-13 14:06:32.316387208 +0600
+++ glibc-2.3.5/localedata/SUPPORTED	2005-08-13 14:10:11.836015152 +0600
@@ -63,6 +63,7 @@
 cy_GB/ISO-8859-14 \
 da_DK.UTF-8/UTF-8 \
 da_DK/ISO-8859-1 \
+da_DK.ISO-8859-15/ISO-8859-15 \
 de_AT.UTF-8/UTF-8 \
 de_AT/ISO-8859-1 \
 de_AT@euro/ISO-8859-15 \
@@ -89,6 +90,7 @@
 en_DK/ISO-8859-1 \
 en_GB.UTF-8/UTF-8 \
 en_GB/ISO-8859-1 \
+en_GB.ISO-8859-15/ISO-8859-15 \
 en_HK.UTF-8/UTF-8 \
 en_HK/ISO-8859-1 \
 en_IE.UTF-8/UTF-8 \
@@ -103,6 +105,7 @@
 en_SG/ISO-8859-1 \
 en_US.UTF-8/UTF-8 \
 en_US/ISO-8859-1 \
+en_US.ISO-8859-15/ISO-8859-15 \
 en_ZA.UTF-8/UTF-8 \
 en_ZA/ISO-8859-1 \
 en_ZW.UTF-8/UTF-8 \
@@ -257,6 +260,7 @@
 ro_RO.UTF-8/UTF-8 \
 ro_RO/ISO-8859-2 \
 ru_RU.KOI8-R/KOI8-R \
+ru_RU.CP1251/CP1251 \
 ru_RU.UTF-8/UTF-8 \
 ru_RU/ISO-8859-5 \
 ru_UA.UTF-8/UTF-8 \
@@ -280,6 +284,7 @@
 sv_FI@euro/ISO-8859-15 \
 sv_SE.UTF-8/UTF-8 \
 sv_SE/ISO-8859-1 \
+sv_SE.ISO-8859-15/ISO-8859-15 \
 ta_IN/UTF-8 \
 te_IN/UTF-8 \
 tg_TJ/KOI8-T \
@@ -293,6 +298,7 @@
 tr_TR/ISO-8859-9 \
 tt_RU.UTF-8/UTF-8 \
 uk_UA.UTF-8/UTF-8 \
+uk_UA.CP1251/CP1251 \
 uk_UA/KOI8-U \
 ur_PK/UTF-8 \
 uz_UZ/ISO-8859-1 \
